export default function query(query) {
  return [
    {
      title: `This is a ${query}`,
      url: `www.LoremIpsum.com`,
      description: "Coordinates: 36°17′56.4″N 30°21′17.3″E﻿ / ﻿36.299000°N 30.354806°E﻿ / 36.299000; 30.354806 Gagae (Greek: Γάγαι), was a town on the southeast coast of Lycia, in what is now the province of Antalya, from which the Gagates lapis derived its name. The ruins are located in Kumluca district, Antalya Province, Turkey.",
      emotion: "😒 10% | 😃 50% | 😱 35% | 🙈 5% | 😡 0%"
    },
    {
      title: `This is a ${query}`,
      url: "www.Dolorsit.com",
      description: "CodeSandbox aims to solve this by allowing developers to simply go to a URL in their browser to start building. This not only makes it easier to get started, it also makes it easier to share. You can just share your created work by sharing the URL, others can then (without downloading) further develop on these sandboxes.",
      emotion: "😒 10% | 😃 50% | 😱 35% | 🙈 5% | 😡 0%"
    },
    {
      title: `${query} is a title`,
      url: "www.amet.com",
      description: "The traditional way to deal with asynchronous calls in JavaScript has been with callbacks. Say we had to make three calls to the server, one after the other, to set up our application. With callbacks, the code might look something like the following (assuming a xhrGET function to make the server call):",
      emotion: "😒 10%& | 😃 50% | 😱 35% | 🙈 5% | 😡 0%"
    },
    {
      title: `${query} is a title`,
      url: `www.LoremIpsum.com`,
      description: "The primary API for a promise is its then method, which registers callbacks to receive either the eventual value or the reason why the promise cannot be fulfilled. Here is a simple “hello world” program that synchronously obtains and logs a greeting. var greeting = sayHello(); console.log(greeting); // 'hello world'.",
      emotion: "😒 10% | 😃 50% | 😱 35% | 🙈 5% | 😡 0%"
    },
    {
      title: `This is a ${query}`,
      url: "www.Dolorsit.com",
      description: "Each promise object will have a then function that can take two arguments, a success handler and an error handler. The success or the error handler in the then function will be called only once, after the asynchronous task finishes. The then function will also return a promise , to allow chaining multiple calls.",
      emotion: "😒 10% | 😃 50% | 😱 35% | 🙈 5% | 😡 0%"
    },
    {
      title: `${query} is a title`,
      url: "www.amet.com",
      description: "Basically, you just pass a promise and it returns promise, but .then from the returned promise can accept a callback function, instead of two arguments as usual - one for result, one for error. Notice that it also works as normal .then if you pass two arguments. <promise> {Promise}; return {Promise}. Example.",
      emotion: "😒 10% | 😃 50% | 😱 35% | 🙈 5% | 😡 0%"
    },
    {
      title: `This is a ${query}`,
      url: `www.LoremIpsum.com`,
      description: "Basic usage is instantiating a dojo/Deferred object, starting the thread and when it completes calling .resolve() with any results and returning the object to the consumer. From a consumer perspective it is generally setting either a .then() callback with potentially an additional error callback as the second argument of then.",
      emotion: "😒 10% | 😃 50% | 😱 35% | 🙈 5% | 😡 0%"
    },
    {
      title: `${query} is a title`,
      url: "www.Dolorsit.com",
      description: "Promises are commonly used with the .then() method. This is a powerful method that allows you to define the callback and errback functions. The first parameter is always the callback and the second is the errback . someAsyncFunction().then(callback, errback);. The callback is invoked once the promise resolves and the",
      emotion: "😒 10% | 😃 50% | 😱 35% | 🙈 5% | 😡 0%"
    },
    {
      title: `This is a ${query}`,
      url: "www.amet.com",
      description: "The then method of the Promise object runs when the promise is completed or rejected, whichever occurs first. If the promise is completed successfully, the fulfillment handler function of the then method runs. If the promise is rejected, the error handler function of the then method (or the catch method) runs.",
      emotion: "😒 10% | 😃 50% | 😱 35% | 🙈 5% | 😡 0%"
    },
    {
      title: `${query} is a title`,
      url: `www.LoremIpsum.com`,
      description: "Basically, you just pass a promise and it returns promise, but .then from the returned promise can accept a callback function, instead of two arguments as usual - one for result, one for error. Notice that it also works as normal .then if you pass two arguments. <promise> {Promise}; return {Promise}. Example.",
      emotion: "😒 10% | 😃 50% | 😱 35% | 🙈 5% | 😡 0%"
    },
  ];
};
